kimchi/src/verifier.rs

```diff
@@ -104,6 +105,7 @@ where
         &self,
         index: &VerifierIndex<G>,
         public_comm: &PolyComm<G>,
+        public: &Vec<G::ScalarField>,
```

Comment:
can this be renamed `public_input`? I think it's clearer

Reply:
Done